UndertaleModLib/Models/UndertaleFont.cs

```diff
@@ -76,8 +87,33 @@ public void Unserialize(UndertaleReader reader)
                 SourceY = reader.ReadUInt16();
                 SourceWidth = reader.ReadUInt16();
                 SourceHeight = reader.ReadUInt16();
-                Shift = reader.ReadUInt16();
-                Offset = reader.ReadUInt32();
+                Shift = reader.ReadInt16();
+                if (reader.undertaleData.GeneralInfo?.Major >= 2 || (reader.undertaleData.GeneralInfo?.Major == 1 && reader.undertaleData.GeneralInfo?.Build == 9999))
+                {
+                    Offset = reader.ReadInt16();
+                    Kerning = reader.ReadUndertaleObject<UndertaleSimpleListShort<GlyphKerning>>();
+                } else
+                {
+                    Offset = reader.ReadInt32(); // Maybe? I don't really know, but this definitely used to work
+                }
+            }
+
+            public class GlyphKerning : UndertaleObject
```

Comment:
So I guess the editor needs an update...
How many entries of this do you usually get? I'm just wondering what controls would be best for this, since now every table row can have subrows...
(and font editing for translators just got harder, ugh)

Reply:
Actually, I feel stupid because I just realized none of the things I've tested on use the list at all. It doesn't really seem necessary to allocate too much space for them (if any).